ts/__tests__/Iterable.spec.ts

```diff
@@ -159,43 +158,43 @@ test("trackEvent_params_methodCalled", () => {
   expect(MockRNIterableAPI.trackEvent).toBeCalledWith(name, dataFields)
 })
 
-test("setAttributionInfo_getAttributionInfo_attributionInfo_returnsAttributionInfo", () => {
-  Iterable.logger.log("setAttributionInfo_getAttributionInfo_attributionInfo_returnsAttributionInfo")
+test('setAttributionInfo_getAttributionInfo_attributionInfo_returnsAttributionInfo', async () => {
+  Iterable.logger.log('setAttributionInfo_getAttributionInfo_attributionInfo_returnsAttributionInfo')
 
   // GIVEN attribution info
-  let campaignId = 1234
-  let templateId = 5678
-  let messageId = "qwer"
+  const campaignId = 1234
+  const templateId = 5678
+  const messageId = 'qwer'
 
   // WHEN Iterable.setAttributionInfo is called with the given attribution info
   Iterable.setAttributionInfo(new IterableAttributionInfo(campaignId, templateId, messageId))
 
   // THEN Iterable.getAttrbutionInfo returns the given attribution info
-  return Iterable.getAttributionInfo().then(attributionInfo => {
+  return await Iterable.getAttributionInfo().then(attributionInfo => {
```

Comment:
Wondering how block functions can turn into async via es lint changes. What made it suggest this change?